diff --git a/dist/cjs/Page/PageCanvas.js b/dist/cjs/Page/PageCanvas.js
index b5e8e1183faadb68e351147a49335069bf84f63a..a19362ba25ea10824d34d4cf62240deb4263f03f 100644
--- a/dist/cjs/Page/PageCanvas.js
+++ b/dist/cjs/Page/PageCanvas.js
@@ -82,8 +82,13 @@ function PageCanvas(props) {
         }
         canvas.width = renderViewport.width;
         canvas.height = renderViewport.height;
-        canvas.style.width = `${Math.floor(viewport.width)}px`;
-        canvas.style.height = `${Math.floor(viewport.height)}px`;
+                
+        const CSS = 96.0;
+        const PDF = 72.0;
+        const PDF_TO_CSS_UNITS = CSS / PDF;
+
+        canvas.style.width = `${Math.floor(viewport.width) * PDF_TO_CSS_UNITS}px`;
+        canvas.style.height = `${Math.floor(viewport.height) * PDF_TO_CSS_UNITS}px`;
         canvas.style.visibility = 'hidden';
         const renderContext = {
             annotationMode: renderForms ? ANNOTATION_MODE.ENABLE_FORMS : ANNOTATION_MODE.ENABLE,
diff --git a/dist/esm/Page/PageCanvas.js b/dist/esm/Page/PageCanvas.js
index 1b72392a3cb99f8b732516e20430a612d17a53d5..294829209a5b4f49ca4d040f5d11366f587f9902 100644
--- a/dist/esm/Page/PageCanvas.js
+++ b/dist/esm/Page/PageCanvas.js
@@ -54,8 +54,13 @@ export default function PageCanvas(props) {
         }
         canvas.width = renderViewport.width;
         canvas.height = renderViewport.height;
-        canvas.style.width = `${Math.floor(viewport.width)}px`;
-        canvas.style.height = `${Math.floor(viewport.height)}px`;
+                
+        const CSS = 96.0;
+        const PDF = 72.0;
+        const PDF_TO_CSS_UNITS = CSS / PDF;
+
+        canvas.style.width = `${Math.floor(viewport.width) * PDF_TO_CSS_UNITS}px`;
+        canvas.style.height = `${Math.floor(viewport.height) * PDF_TO_CSS_UNITS}px`;
         canvas.style.visibility = 'hidden';
         const renderContext = {
             annotationMode: renderForms ? ANNOTATION_MODE.ENABLE_FORMS : ANNOTATION_MODE.ENABLE,
